src/Filter/StringFilter.php

```diff
@@ -79,6 +79,13 @@ public function filter(ProxyQueryInterface $queryBuilder, $alias, $field, $data)
                     break;
                 default:
                     $format = $this->getOption('format');
+
+                    if ('%%%s%%' !== $format) {
+                        @trigger_error(
+                            'The format option is deprecated since sonata-project/doctrine-orm-admin-bundle 3.x and will be removed in version 4.0.',
```

Comment:
```suggestion
                            'The "format" option is deprecated since sonata-project/doctrine-orm-admin-bundle 3.x and will be removed in version 4.0.',
```